core/src/test/java/org/infinispan/replication/ForceSyncAsyncFlagsTest.java

```diff
@@ -55,9 +52,7 @@ public void testForceAsyncFlagUsage() throws Exception {
 
       // check that the replication call was sync
       cache1.put("k", "v");
-      verify(mockTransport)
-            .invokeCommandOnAll(any(ReplicableCommand.class), any(ResponseCollector.class), any(DeliverOrder.class),
-                                anyLong(), any(TimeUnit.class));
+      verify(mockTransport).invokeCommandOnAll(any(), any(), any(), any(), anyLong(), any());
```

Comment:
why have you removed the classes?

Reply:
I just find it more readable when the method call fits into one line, and the test only cares about the name of the method being called (`invokeCommandOnAll` vs. `sendToAll`).